### DISCUSSION TITLE
Any way to manually import LastListen from json file for each book?

### DISCUSSION THREAD

**Discussion by ragnars-stuff** [advplyr/audiobookshelf#3261] [Q&A]
I working on moving my library to ABS, anyone tried to import the lastlisten data?

Used Libation to export my library and it inclued data from Audible with last listen.
Would love for ASB to import the data, if not then I could write a script of some sort to import it into the ASB database.

**Comment by nichwall** [comment]
ABS currently only supports one listen per book (so if you want to listen to the book again you need to clear the progress for your user). There is a feature request to keep track of read/listen count, but not sure if/when that will be added. 

To import your "last time listened" information, you can either add the progress information directly to the database or through the API using a custom script.
